README.md

```diff
@@ -1,11 +1,64 @@
 # Description
 Coforma's response to the SWIFT technical challenge
 
-## Local Setup
+## TODO: Table of Contents
+
+- TODO: Links here
+
+## Getting Started
+
+### One time only
+
+Before developing locally, you'll need to install some tools. You may have some of these already, or you may not.
+
+- Install [nvm](https://github.com/nvm-sh/nvm):
+    ```bash
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+    ```
+
+- Use nvm to install specified version of [node](https://nodejs.org/en):
+    ```bash
+    cd <project-directory>
+    nvm use
+    ```
+- Install [yarn](https://classic.yarnpkg.com/en/docs/install):
+
+    ```bash
+    brew install yarn
+    ```
+
+- Install [pre-commit](https://pre-commit.com/#install):
+
```

Comment:
This is the wrong command. I thinks its pre-commit

Reply:
This comment should have been against the next line

Reply:
Ah yep. Typo. Will fix.